azure_functions_worker/main.py

```diff
@@ -26,6 +26,17 @@ def parse_args():
                              'syslog, or a file path')
     parser.add_argument('--grpcMaxMessageLength', type=int,
                         dest='grpc_max_msg_len')
+    parser.add_argument('--functions-uri', dest='functions_uri',
+                        help='URI with IP Address and Port used to'
```

Comment:
nit: type=str for these args

Reply:
any of these fields are required? default length for max_msg_len arg?

Reply:
Updated type=str for args. These args are required to handle from worker side. Thanks